src/components/ViewAgents/SearchBarWizards.tsx

```diff
@@ -0,0 +1,133 @@
+import { useState, useRef } from "react";
+import { Form, Button } from "react-bootstrap";
+import { Link, useNavigate } from "react-router-dom";
+import classNames from "classnames";
+
+import { WizardDetailsBasicWithTimeStamp } from "declarations/wizard_details/wizard_details.did";
+
+import CloseIcon from "src/assets/close_icon.svg?react";
+import SearchIcon from "src/assets/search_icon.svg?react";
+
+function SearchBarWizards(
+    {popularWizards, setSearchButtonActive, suggestionResults, setSuggestionResults}: 
+    {
+        popularWizards: WizardDetailsBasicWithTimeStamp[],
+        setSearchButtonActive: React.Dispatch<React.SetStateAction<boolean>>
+        suggestionResults: WizardDetailsBasicWithTimeStamp[]
+        setSuggestionResults: React.Dispatch<React.SetStateAction<Array<WizardDetailsBasicWithTimeStamp>>>
+    }
+) {
+
+    const navigate = useNavigate();
+    
+    const [suggestionActive, setSuggestionActive] = useState(false);
+    const [suggestionIndex, setSuggestionIndex] = useState<number>(-1);
```

Comment:
```suggestion
    const [suggestionIndex, setSuggestionIndex] = useState(-1);
```

Reply:
Changing it

Reply:
https://github.com/elna-ai/ELNA-DApp/pull/204/commits/7888d39dd7acb2b4b2f4d08148e3161aa6150055